### DISCUSSION TITLE
Does apisix support to prevent from XSS attack?

### DISCUSSION THREAD

**Discussion by shfpflyawei** [apache/apisix#5975] [Q&A]
apisix可有插件 能够避免存储型XSS漏洞，避免跨站脚本攻击

**Comment by shfpflyawei** [comment]
Cross SiteScript

**Reply by tzssangglass** [reply]
cors plugin?

**Reply by juzhiyuan** [reply]
Hi @shfpflyawei, AFAIK, we could use Apache APISIX's cors plugin to configure Cross Origin Resource Sharing (CORS). As for XSS, according to https://github.com/apache/apisix/issues/1868, it seems that we don't support yet 🤔 need @tzssangglass's confirmation.

**Reply by tzssangglass** [reply]
I am not very familiar.

**Reply by juzhiyuan** [reply]
Thanks anyway, and cc @bzp2010 😄 Do you know about this?
